README.md

```diff
@@ -574,6 +611,59 @@ expect(myComponent).toContainHtml('<span>Hello world!</span>');
 
 ## FAQ
 
+### Can I use this with a setup that aliases preact to react?
+
+The complete API compatibility with `@shopify/react-testing` means you can do the same kind of module aliasing you can do with `react` and `preact` with `@shopify/react-testing` and `@shopify/preact-testing`. If you are using jest, you would:
+
+1. Install `@shopify/preact-testing`
+
+```
+$ yarn add @shopify/preact-testing
+```
+
+2. Add a `moduleNameMapper` object to your `jest.config.js` config with the following keys
+
+```jsx
+moduleNameMapper: {
+  '^react$': 'preact/compat',
+  '^@shopify/react-testing$': '@shopify/preact-testing',
+  '^react-dom$': 'preact/compat',
+  '^react-dom/test-utils$': 'preact/test-utils',
+}
+```
+
+3. Create a `setupFilesAfterEnv` entry and setup the testing library there
+
+```tsx
+import 'preact/debug';
+import '@shopify/preact-testing/matchers';
+import {destroyAll} from '@shopify/preact-testing';
+
+// eslint-disable-next-line jest/require-top-level-describe
+afterEach(() => {
+  destroyAll();
+});
+```
```

Comment:
won't you need to do this for all the cases? a little confuse that this is in the `aliases preact to react` section only

Reply:
you do need to do this in all cases, I just wanted to cover the steps that it takes to add the library to a react app

Reply:
Maybe we can move this into the Basic setup section? Most of these setup steps are great to have, I just worry having it in a FAQ would implemented you only need to do this when you are doing aliases.

Reply:
yeah we can move this to basic usage I suppose